debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/actions/ActionMessages.properties

```diff
@@ -46,6 +46,11 @@ OpenBreakpointMarkerAction_Go_to_File_for_Breakpoint_2=Go to File for Breakpoint
 RelaunchActionDelegate_An_exception_occurred_while_launching_2=An exception occurred while launching
 RelaunchActionDelegate_Launch_Failed_1=Launch Failed
 
+EnableAllBreakpointsAction_0=Enable All Breakpoints
+EnableAllBreakpointsAction_1=Enable All Breakpoints?
```

Comment:
what is the "?" for?

Reply:
<img width="707" alt="image" src="https://github.com/user-attachments/assets/7b2486c3-63a6-4c01-938b-d90e45dc6fc7">

Similar to disable all there is a confirmation popup, for that it is used.

Reply:
Why does it make any sense to ask that double check? It's not anything bad. i would prefer to just to it.

Reply:
Already implemented Disable All Breakpoints options do the similar check prior its operation. so did the same. should I then remove the confirmation part or keep it as it is now ?

Reply:
As you like. You will use the action so you probably know best if there is any use of such question.

Reply:
Just for a reminder if accidental click is occurred. can be disabled if user ticks do not ask me